pyiron/base/generic/parameters.py

```diff
@@ -471,9 +471,9 @@ def set_value(self, line, val):
             new_array.append(val)
             new_comments.append("")
             new_params.append("")
-            new_array = np.array(new_array)
-            new_comments = np.array(new_comments)
-            new_params = np.array(new_params)
+            new_array = np.array(new_array).tolist()
```

Comment:
Is this part related to the kpoints? Otherwise I would suggest we move this to a separate pull request, because I am not yet sure if it might break some functionality elsewhere.

Reply:
We can create a separate pull request. @FeLoch, shall I do this?

Reply:
please, go ahead!

Reply:
Done here https://github.com/pyiron/pyiron/pull/446.

Reply:
@FeLoch Is this change necessary for your other changes to work?

Reply:
yes it is. Without these changes the file would be converted in an array, if you increased the number of lines compared to the default file. Therefore, those changes are only necessary for line mode calculations